.github/workflows/ci.yaml

```diff
@@ -0,0 +1,121 @@
+name: CI
+on: [push, pull_request]
+jobs:
+    phpstan:
+        name: PHPStan
+        runs-on: ubuntu-20.04
+        strategy:
+            matrix:
+                php-version: [7.3]
```

Comment:
buď bych nechal stejně (je to "Use github actions", tak dělat tu zárveň update je meh), nebo bumpnul na nejnovjěší, ne? tedy 7.4

nebo ti šlo o to testovat nejnižší možnou verzi?

Reply:
btw používat metrix když tu máš jen jednu verzi mi přijde zbytečný

Reply:
umm trochu jsem kopiroval tvoje :D

Reply:
s tím matrix si sám nejsem jistý co je lepší :D tohle je "zbytečně verbose", na druhou stranu snažší na rozšíření, kdyby ses rozhodl pouštět to na více verzích :D

Reply:
matrix necham, je pak lepsi maintenance tech ostatnich steps, ktery jsou diky stejny

Reply:
cs i phpstan chces spis na ty nejnizsi poustet

Reply:
done